src/Cli/dotnet/CommonOptions.cs

```diff
@@ -129,7 +129,7 @@ public static CliArgument<string> DefaultToCurrentDirectory(this CliArgument<str
             {
                 Description = CommonLocalizableStrings.DisableBuildServersOptionDescription
             }
-            .ForwardAsMany(_ => new string[] { "-p:UseRazorBuildServer=false", "-p:UseSharedCompilation=false", "/nodeReuse:false" });
+            .ForwardAsMany(_ => ["--property:UseRazorBuildServer=false", "--property:UseSharedCompilation=false", "/nodeReuse:false"]);
```

Comment:
For testing purposes, since we're using the shared `Property` option, these output properties to forward to MSBuild are all unified to `--project` for maximal clarity.